src/accelerate/accelerator.py

```diff
@@ -2601,7 +2601,7 @@ def pad_across_processes(self, tensor, dim=0, pad_index=0, pad_first=False):
         """
         return pad_across_processes(tensor, dim=dim, pad_index=pad_index, pad_first=pad_first)
 
-    def unwrap_model(self, model, keep_fp32_wrapper: bool = True):
+    def unwrap_model(self, model, keep_fp32_wrapper: bool = True, keep_torch_compile: bool = False):
```

Comment:
I think you wanted to set keep_torch_compile to True for a couple of version @muellerzr ?

Reply:
Correct, let's default to `True` until 2.5.0, to give users ~3 months. 

Essentially we should default to `None`, and then if it gets `None` warn that the default to this will be changing

Reply:
I've defaulted `keep_torch_compile=True` in the latest commit.